### DISCUSSION TITLE
How to get OG Metadata to display translated title?

### DISCUSSION THREAD

**Discussion by DavidKrassnig** [george-gca/multi-language-al-folio#2] [General]
Hello all,
I'm very happy with this multilingual variant of al-folio (thanks for making it). I've noticed an issue, however: when I set serve_og_meta in _config.yml to true, the og:title in the HTML source code is set to the variable name instead of the variable's value.

E.g., this is the current output:
`<meta property="og:title" content="David Krassnig | titles.about">`

When this would be the desired output:
`<meta property="og:title" content="David Krassnig | About me">`

Unfortunately, I'm not well-versed in these matters. Anyone have an idea on how to fix this? I assume the responsible file to be modified is _includes/metadata.html, but I'm not entirely certain.

Side note: This issue (or its solution) may or may not be connected to the title not correctly displaying in the example news "A long announcement with details"

I'd be happy to help, if anyone could point me in the right direction.

**Comment by george-gca** [comment]
Hi @DavidKrassnig, just now implemented this feature. I struggled with it for a while, but now I made it work.

**Reply by DavidKrassnig** [reply]
Hi @george-gca , thanks, you are awesome!
